x-pack/plugins/alerting/server/saved_objects/schemas/raw_rule/v1.ts

```diff
@@ -167,9 +167,10 @@ const rawRuleAlertsFilterSchema = schema.object({
             isMultiIndex: schema.maybe(schema.boolean()),
             type: schema.maybe(schema.string()),
             key: schema.maybe(schema.string()),
-            params: schema.maybe(schema.recordOf(schema.string(), schema.any())), // better type?
+            params: schema.maybe(schema.any()),
```

Comment:
I'm not sure about this one, shouldn't we replicate the type instead of opening up? https://github.com/elastic/kibana/blob/7d29cb1a77f9a0c284d0b66d723d054460b512e7/packages/kbn-es-query/src/filters/build_filters/types.ts#L84

Reply:
@cnasikas We talked about it. But you see, there is the type in `kbn-es-query` package. Maybe we can import and use it. Wdyt?

Reply:
IMO, we should not persist with the `filters` as the schema is only related to the UI and is not needed for the backend or a consumer of the API. I am not sure we can efficiently catch up with all variants of the `params` schema. An alternative would be to do `schema.maybe(schema.oneOf([schema.recordOf(schema.string(), schema.any()), schema.arrayOf(schema.recordOf(schema.string(), schema.any())]))` but maybe it is not worth the effort. What do you both think?

Reply:
Sound reasonable!